Utils/Dataflow/009_oracleConnector/dbConnection.py

```diff
@@ -210,3 +215,6 @@ def results(self, qname, arraysize=1000, rows_as_dict=False):
                 else:
                     yield row
             results = c.fetchmany(arraysize)
+
+        rc2 = c.rowcount
+        sys.stderr.write("(TIMING) (%s) (n_rows): %s\n" % (qname, rc2 - rc1))
```

Comment:
Pardon my possible stupidity, but why is change in amount of rows logged as TIMING?

Reply:
@Evildoor, good question, indeed ))
How long the query execution took ([first record logged as TIMING](https://github.com/PanDAWMS/dkb/pull/162/commits/331e6f1ef81add3099b559835760d58575537596#diff-410871e02bb7fd9f1844d212d852fbe7R158)) makes no sense by itself -- we need to know, how many records were retrieved via this query. It is obvious that querying all tasks for 1 month will take longer than that for 1 hour, right? So the actual metric of interest here is seconds per row, not seconds or rows themselves.

You are right, it was not the most accurate decision to get the metrics like this, marking them all as `(TIMING)`, but if we want to change it -- it is a matter for another PR (as this change came form `master`).

Reply:
Fair enough.